clientlib/src/main/proto/yelp/nrtsearch/search.proto

```diff
@@ -210,6 +210,19 @@ message ExistsQuery {
     string field = 1; // Field in the document to query
 }
 
+enum CompletionQueryType {
+    PREFIX_QUERY = 0; // PrefixCompletionQuery within ContextCompletionQuery
+    FUZZY_QUERY = 1; // FuzzyCompletionQuery within ContextCompletionQuery
+}
+
+// A query used for suggest completion
+message CompletionQuery {
+    string field = 1; // Field in the document to query, must be a SuggestField
+    CompletionQueryType queryType = 2;
+    string text = 3;
+    repeated string contexts = 4;
+}
```

Comment:
nit: add remain field documentation, also it needs to be on the line above the field to generation swagger docs correctly (yes, it is wrong in a lot of other places)